network-api/networkapi/templates/donate/pages/base.html

```diff
@@ -28,12 +28,6 @@
 
 {% block bodyclass %} donate tw-m-0 tw-font-sans tw-font-normal tw-text-base tw-font-normal tw-text-black tw-bg-white tw-scroll-smooth {% endblock %}
 
-{% block donate_banner %}
```

Comment:
This block has been deleted, as I have confirmed with Genevieve that Donate page's are OK to also render the donate banner, if it has been set. 

Since this template already extends from `pages/base.html`, this block is no longer needed!